sys/reaper/reaper_unix.go

```diff
@@ -107,3 +108,59 @@ func (m *Monitor) Unsubscribe(c chan runc.Exit) {
 	close(c)
 	m.Unlock()
 }
+
+func (m *Monitor) getSubscribers() map[chan runc.Exit]struct{} {
+	out := make(map[chan runc.Exit]struct{})
```

Comment:
`len(m.subscribers)` (in the lock)

Reply:
I guess if this is never expected to be large it really doesn't matter and maybe better than allocating while holding the lock.